min metadata schema.json

```diff
@@ -15,7 +16,8 @@
           "type": "string"
         },
         "Email": {
-          "type": "string"
+          "type": "string",
+          "description": "required so we can notify you when your dataset is processed"
```

Comment:
Please don't use second person, as the descriptions might be shown to people other than the uploader.